src/puptoo/mq/consume.py

```diff
@@ -33,5 +33,5 @@ def init_consumer():
 
     consumer.subscribe([config.ADVISOR_TOPIC,
                         config.COMPLIANCE_TOPIC,
-                        config.MALWARE_DETECTION_TOPIC])
+                        config.ANNOUNCE_TOPIC])
```

Comment:
Regardless of service headers, all messages are being sent on `platform.upload.announce` topic by ingress. With this change, it is going to process messages with `advisor & compliance` service headers twice.